packages/nextjs/app/builders/0x872f1fFdeEe35a6074658B4FB185E2582Db7896A/page.tsx

```diff
@@ -0,0 +1,128 @@
+import Head from "next/head";
+import Image from "next/image";
+import { NextPage } from "next";
+import {
+  AcademicCapIcon,
+  CodeBracketIcon,
+  RocketLaunchIcon,
+  StarIcon,
+  TrophyIcon,
+  XMarkIcon,
+} from "@heroicons/react/24/outline";
+import { BuidlGuidlLogo } from "~~/components/assets/BuidlGuidlLogo";
+import { Address } from "~~/components/scaffold-eth";
+
+const CleanerPage: NextPage = () => {
+  return (
+    <>
+      <Head>
+        <title>Cleaner - Web3 Enthusiast and Developer</title>
+        <meta
+          name="description"
+          content="Official page of Cleaner, a full-stack and smart contract developer passionate about Web3, hackathons, and innovative technology like ZK and AI."
+        />
+        <meta property="og:title" content="Cleaner - Web3 Enthusiast and Developer" />
+        <meta
+          property="og:description"
+          content="Discover the projects and passions of Cleaner, including development in Web3 and participation in global hackathons."
+        />
+        <meta property="og:image" content="/thumbnail.jpg" />
+        <meta property="og:type" content="website" />
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta name="twitter:title" content="Cleaner - Web3 Enthusiast and Developer" />
+        <meta
+          name="twitter:description"
+          content="Learn more about Cleaner's work in blockchain technology and his contributions to decentralized applications."
+        />
+        <meta name="twitter:image" content="/thumbnail.jpg" />
+      </Head>
```

Comment:
Lol sorry this is last thing! Instead of using Head tag which is deperecated in app router 

![Screenshot 2024-05-16 at 1 10 05 PM](https://github.com/BuidlGuidl/batch5.buidlguidl.com/assets/80153681/c8b4fa8e-cd7c-4774-b0e3-03841f7ebc56)

Your could use nice utility which Scaffold ETH provides [`getMetadata`](https://github.com/BuidlGuidl/batch5.buidlguidl.com/blob/24bbf023c27b00c1b09233aff4596a8d1a0e0eec/packages/nextjs/utils/scaffold-eth/getMetadata.ts) which you can use directly by just passing title, description and res others things are handled internally 🙌

Checkout the example here https://github.com/BuidlGuidl/batch5.buidlguidl.com/blob/24bbf023c27b00c1b09233aff4596a8d1a0e0eec/packages/nextjs/app/debug/page.tsx#L5